cli/core/test/e2e/framework/plugin_lifecycle_operations.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
nit: I missed this during the previous review but the YEAR mentioned here is incorrect. We should use the current year for all new files that we add.

Reply:
updated